util/install/debian.sh

```diff
@@ -15,8 +15,14 @@ _qmk_install() {
         build-essential clang-format diffutils gcc git unzip wget zip \
         python3-pip binutils-avr gcc-avr avr-libc binutils-arm-none-eabi \
         gcc-arm-none-eabi libnewlib-arm-none-eabi avrdude dfu-programmer \
-        dfu-util teensy-loader-cli libhidapi-hidraw0 libusb-dev \
-        picolibc-riscv64-unknown-elf gcc-riscv64-unknown-elf binutils-riscv64-unknown-elf
+        dfu-util teensy-loader-cli libhidapi-hidraw0 libusb-dev
 
-    python3 -m pip install --user -r $QMK_FIRMWARE_DIR/requirements.txt
+    # RISC-V toolchains with picolibc support are only available from Debain 11+ and Ubuntu 21.04+ onwards.
+    if [ "$(apt-cache show picolibc-riscv64-unknown-elf gcc-riscv64-unknown-elf binutils-riscv64-unknown-elf > /dev/null 2>&1)" ]; then
```

Comment:
Can't really test this at the moment, but it looks like the condition as written will always be false; was the exit code check intended instead?
```suggestion
    if apt-cache show picolibc-riscv64-unknown-elf gcc-riscv64-unknown-elf binutils-riscv64-unknown-elf > /dev/null 2>&1; then
```

Reply:
You are right, although the `apt-cache show` mechanism doesn't work if there is one valid package in the check e.g. `apt-cache show xqäü curl > /dev/null 2>&1; echo $?` will have an return code of 0. Therefore I use `apt-get install --simulate` which works as expected.

Reply:
Done.